app/grants/templates/grants/detail/index.html

```diff
@@ -67,7 +68,7 @@
       <div id="side-cart" class="grant-side-cart px-5 px-md-4 pb-5 pb-md-0 pt-md-0" style="display: none;">
         {% include 'grants/detail/side-cart.html' %}
       </div>
-      </div>
+      </div> <!-- meh? -->
```

Comment:
What was this comment for? 😁

Reply:
Good catch! Noticed this extra `div` tag at some point and wanted to double check it, then forgot. It's not needed & is now removed.